components/expand-collapse/README.md

```diff
@@ -1,36 +1,76 @@
-# Expand Collapse
+# Collapsible Containers
 
-## Expand Collapse Content
+## Expand Collapse Content [d2l-expand-collapse-content]
 
 The `d2l-expand-collapse-content` element can be used to create expandable and collapsible content. This component only provides the logic to expand and collapse the content; controlling when and how it expands or collapses is the responsibility of the user.
 
+<!-- docs: start hidden content -->
 ![Expand Collapse Content](./screenshots/expand-collapse-content.gif?raw=true)
+<!-- docs: end hidden content -->
 
+<!-- docs: demo live name:d2l-expand-collapse-content autoSize:false display:block size:small -->
 ```html
 <script type="module">
+  import '@brightspace-ui/core/components/button/button.js';
   import '@brightspace-ui/core/components/expand-collapse/expand-collapse-content.js';
-</script>
 
-<d2l-expand-collapse-content expanded>
+  const button = document.querySelector('d2l-button');
+  button.addEventListener('click', () => {
+    const section = document.querySelector('d2l-expand-collapse-content');
```

Comment:
Maybe a good spot to try out `<!-- docs: hidden content-->`?

Reply:
I think I'll actually leave this one as is for now since it does advice on how to actually implement.